src/test/run-pass/issue-3121.rs

```diff
@@ -8,7 +8,7 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-// pretty-expanded FIXME #23616
+// no-pretty-expanded FIXME #23616 assert_receiver_is_clone
```

Comment:
Instead of `no-pretty-expanded` I think you can just remove these directives altogether